### DISCUSSION TITLE
iOS widget keep asking for "Tap to sign in"

### DISCUSSION THREAD

**Discussion by smol-ninja** [community/community#68942] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Github iOS widget cannot detect the app login and keep showing "Tap to sign in".

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by seriouslag** [comment, marked as answer]
Right-click widget.
Click "Edit "Github""
Select Choose on the account banner.
Select account.
Click "Done"

It still shows "Tap to sign in" if you click the widget. Widget needs work
Audible widget does the same

**Reply by gabrielmocanu** [reply]
This works great on home screen, but do you know any solution for the Lock Screen?
I can't find a way to edit GitHub widget on lock screen.
